.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules/
 /.project
+/.vscode
+package-lock.json
+yarn.lock
```

Comment:
Remove these changes; they're not related to the issue at hand.

Reply:
Done.